notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -405,18 +407,8 @@
     }
    },
    "source": [
-    "### Run training pipeline\n",
-    "Tune floating-point ResNet-18 on Tiny-ImageNet-200 to obtain a pre-trained model for quantization."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "118rlV22_PB5"
-   },
-   "source": [
-    "> **NOTE** The model is not tuned till the final accuracy. For the sake of simplicity of the tutorial, we tune for 4 epoch only and take the last model state for quantization.\n",
-    "\n"
+    "### Get a pretrained FP32 model\n",
+    "А pre-trained floating-point model is a prerequisite for quantization. It can be obtained by tuning from scratch  with the code below. However, usually it takes a lot of time. Therefore, we had already run this code and received the good enough weights after 4 epochs (for the sake of simplicity don't tune until the best accuracy). By default, this notebook just loads this weights without launching training. To tune model from scratch set `pretrained=False`.\n"
```

Comment:
```suggestion
    "А pre-trained floating-point model is a prerequisite for quantization. It can be obtained by tuning from scratch  with the code below. However, this usually takes a lot of time. Therefore, we have already run this code and received good enough weights after 4 epochs (for the sake of simplicity, we did not tune until the best accuracy). By default, this notebook just loads these weights without launching training. To tune the model from scratch set `pretrained=False`.\n"
```